core/types/gen_transaction_conditional_json.go

```diff
@@ -33,11 +34,13 @@ func (t TransactionConditional) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON unmarshals from JSON.
 func (t *TransactionConditional) UnmarshalJSON(input []byte) error {
 	type TransactionConditional struct {
-		KnownAccounts  *KnownAccounts  `json:"knownAccounts"`
-		BlockNumberMin *hexutil.Big    `json:"blockNumberMin,omitempty"`
-		BlockNumberMax *hexutil.Big    `json:"blockNumberMax,omitempty"`
-		TimestampMin   *hexutil.Uint64 `json:"timestampMin,omitempty"`
-		TimestampMax   *hexutil.Uint64 `json:"timestampMax,omitempty"`
+		KnownAccounts  *KnownAccounts        `json:"knownAccounts"`
```

Comment:
There should be a way to have the autogen tool output this. We shouldn't merge a diff to an autogenerated file.

Update the following struct:
```
type transactionConditionalMarshalling struct {
	BlockNumberMax *hexutil.Big
	BlockNumberMin *hexutil.Big
	TimestampMin   *hexutil.Uint64
	TimestampMax   *hexutil.Uint64
}
```

Like so:

```
type transactionConditionalMarshalling struct {
	BlockNumberMax *math.HexOrDecimal256
	BlockNumberMin *math.HexOrDecimal256
	TimestampMin   *math.HexOrDecimal256
	TimestampMax   *math.HexOrDecimal256
}
```

Reply:
ah yea good point! updated